reference/authentication/kubernetes-verify-service-account.mdx

```diff
@@ -1,3 +0,0 @@
----
-openapi: "POST /auth/v1/method/kubernetes/serviceaccount"
```

Comment:
Did you mean to remove this?

Reply:
I did but I can add it back. It seemed like the Authentication section was a bit crowded with endpoints that a user might not use directly (like the OIDC ones).

Reply:
Yeah we can drop it if we're not concerned with documenting it.
Just checking it was intentional.